db/migrate/20230425093411_create_foods.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :foods do |t|
+      t.string :name
+      t.string :measurement_unit
+      t.decimal :price
+      t.integer :quantity
+      t.integer :user_id
+
+      t.timestamps
+    end
```

Comment:
- [ ] The`user_id` is a foreign key therefore it is best practice to make sure it is indexed because indexing makes columns faster to query 👍